StoreSpidLogs/__test__/index.test.ts

```diff
@@ -13,9 +12,8 @@ import { IPString } from "italia-ts-commons/lib/strings";
 import { aFiscalCode } from "../../__mocks__/mocks";
 import { index, SpidBlobItem, SpidMsgItem } from "../index";
 
-const today = format(new Date(), "YYYY-MM-DD");
```

Comment:
why is this change necessary here ?

Reply:
It's an optimization, trying to not repeat `new Date()` each time we have to instantiate a Date for testing purposes

Reply:
I just see a line swapped (I don't see any optimization in this diff) 
anyway it's out of the scope of this pr, can we revert this file to the version from master ?

Reply:
done!